### PR TITLE
 Add basic MachineDisruptionBudget e2e tests

### PR DESCRIPTION
**Pull request by cynepco3hahue**
Checks the status of MDB object with healthy and unhealthy machine.

### CODE REVIEW

**Comment by cynepco3hahue** [comment]
@ingvagabund heya, I can see that https://gcsweb-ci.svc.ci.openshift.org/gcs/origin-ci-test/pr-logs/pull/openshift_cluster-api-actuator-pkg/83/pull-ci-openshift-cluster-api-actuator-pkg-master-e2e-aws-operator/276/artifacts/release-latest/release-payload-latest/ lacks MDB CRD but under the vendor, it exists.
Do you know how can I add it?

File: Makefile [hunk 1]
```diff
@@ -50,7 +50,7 @@ test-e2e: ## Run openshift specific e2e test
 	# Feature:Operator tests remove deployments. Thus loosing all the logs
 	# previously acquired.
 	hack/ci-integration.sh -ginkgo.v -ginkgo.noColor=true -ginkgo.focus "Feature:Operators"
-	hack/ci-integration.sh -ginkgo.v -ginkgo.noColor=true -ginkgo.skip "Feature:Operators|TechPreview"
+	# hack/ci-integration.sh -ginkgo.v -ginkgo.noColor=true -ginkgo.skip "Feature:Operators|TechPreview"
```
Comment (enxebre) [inline Makefile]: can we remove commented out code?

Comment (cynepco3hahue) [reply Makefile]: yes it only to run tests, I will remove the last commit

**Comment by cynepco3hahue** [comment]
For the reference - https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_cluster-api-actuator-pkg/83/pull-ci-openshift-cluster-api-actuator-pkg-master-e2e-aws-operator/323

File: Makefile (hunk 1 above)
Comment (cynepco3hahue) [reply Makefile]: done

**Comment by cynepco3hahue** [comment]
Currently, this PR does not relevant anymore, because moving part of the MDB logic under the MHC controller.

**Pull request closed without merging**